src/hooks/pageHooks/utils.js

```diff
@@ -0,0 +1,21 @@
+export function extractPageInfo({ data, pageData, resourceRoomName }) {
```

Comment:
not entirely sure if this behaviour is identical to the old one as the `if (!data) return pageData` seems to be additional

Reply:
This is for edits within editPage - since both are routed through this point now, we need some way to bypass this method